NGitLab.Mock/UserCollection.cs

```diff
@@ -63,5 +63,58 @@ internal IEnumerable<User> SearchByUsername(string username)
         {
             return this.Where(user => string.Equals(user.UserName, username, StringComparison.OrdinalIgnoreCase));
         }
+
+        internal IEnumerable<User> Get(UserQuery query)
+        {
+            var users = this.AsQueryable();
+
+            if (query.IsActive.HasValue && query.IsActive.Value)
```

Comment:
You can make the condition simpler and clearer as follows:
```suggestion
            if (query.IsActive == true)
```

Same thing for other following conditions...

Reply:
Updated